docs/logger.rst

```diff
@@ -137,3 +137,59 @@ You can use predefined celery task ``security.tasks.call_django_command`` to run
     from security.tasks import call_django_command
 
     call_django_command.apply_async(args=('check',))
+
+.. class:: security.tasks.LoggedTask
+
+  There are several methods of ``LoggedTask`` which you can use for your advanced tasks logic.
+
+  .. property:: task_log
+
+    The method returns instance o CeleryTaskLog related with your task.
+
+  .. method:: on_apply_task(task_log, args, kwargs, options)
+
+    The method is called before task is queued. You can override method.
+
+  .. method:: on_start_task(task_log, args, kwargs)
+
+    The method is called when if the task was started.
+
+  .. method:: on_success_task(task_log, args, kwargs, retval)
+
+    The method is called when if the task was successfully completed.
+
+  .. method:: on_failure_task(task_log, args, kwargs, exc)
+
+    The method is called when if the task raised an exception and is not retried.
+
+  .. method:: on_retry_task(task_log, args, kwargs, exc)
+
+    The method is called when if the task raised an exception and is retried.
+
+  .. property:: default_retry_delays
+
+    Similar like celery ``default_retry_delay`` which you can use to define how log will be retried task waiting, property ``default_retry_delays`` can be used for defining the same but every task attempt can have set different delay::
+
+        @celery_app.task(
+            base=LoggedTask,
+            bind=True,
+            name='retry_task',
+            autoretry_for=(RuntimeError,),
+            default_retry_delays=(1 * 60, 5 * 60, 10 * 60, 30 * 60, 60 * 60))
+        def retry_task(self):
+            ...
+
+    The ``retry_task`` will be retried on 1 minute for second attempt, 5 minutes for third attempt and so on.
+
+  .. property:: stale_time_limit
+
+    ``stale_time_limit`` it value in seconds which defines, how long will take time to be task set as expired. Default value can be set with ``CELERYD_TASK_STALE_TIME_LIMIT`` in Django settings.
```

Comment:
``stale_time_limit`` is value in seconds which defines, how long it will take to set the task as expired.